common/src/main/java/io/github/gaming32/bingo/client/ClientIconTooltip.java

```diff
@@ -19,8 +20,8 @@ public int getWidth(Font font) {
     }
 
     @Override
-    public void renderImage(Font font, int x, int y, GuiGraphics guiGraphics) {
+    public void renderImage(Font font, int x, int y, int width, int height, GuiGraphics guiGraphics) {
         Minecraft.getInstance().getTextureManager().getTexture(tooltip.icon()).setFilter(true, false);
-        guiGraphics.blit(tooltip.icon(), x, y, SIZE, SIZE, 0f, 0f, SIZE, SIZE, SIZE, SIZE);
+        guiGraphics.blit(RenderType::guiTextured, tooltip.icon(), x, y, SIZE, SIZE, 0, 0, SIZE, SIZE, SIZE, SIZE);
```

Comment:
There is an overload to simplify this:
```suggestion
        guiGraphics.blit(RenderType::guiTextured, tooltip.icon(), x, y, SIZE, SIZE, 0, 0, SIZE, SIZE);
```